blank_react_project/README.md

```diff
@@ -0,0 +1,60 @@
+<br />
+<br />
+
+<p>
+<img src="https://nearprotocol.com/wp-content/themes/near-19/assets/img/logo.svg?t=1553011311" width="240">
+</p>
+
+<br />
+<br />
+
+## Template for NEAR dapps
+### Requirements
+##### IMPORTANT: Make sure you have the latest version of NEAR Shell and Node Version > 10.x 
+1. node and npm
+2. near shell
+install with 
+```
+npm i -g near-shell
+```
+3.(optional) install yarn to build
+```
+npm i -g yarn
+```
+
+### Features
+* Create NEAR dapps with a React frontend 🐲
+* We got Gulp! 💦
+* We got Corgis? [🐶](https://corgis.nearprotocol.com) 
+
+### To run on testnet
+Step 1: Create account for the contract and deploy the contract.
+In the terminal
+```
+near login
+```
+click the link and create your own contract ID
+
+Step 2:
+modify src/config.js line that sets the contractName. Set it to id from step 1.
+```
+const CONTRACT_NAME = "contractId"; /* TODO: fill this in! */
+```
+Step 3:
+deploy the smart contract, run the command:
+```
+npm run deploy:contract
+```
+Step 4:
+Finally, run the command in your terminal.
+```
+npm(yarn) intall && npm(yarn) start
```

Comment:
it's better to have copy-pastable commands, i.e. in this case if you want to provide alternate just spell them out separately like this:

```
yarn && yarn start
```

```
npm install && npm run start
```

Reply:
though I'd prefer we just stick to `yarn` everywhere to avoid confusion, unless someone has objections

Reply:
sorry not get any notification until today I check it myself. I will change it

Reply:
for using fonts/colors from brand, where I can find the asserts?

Reply:
change read me with separately copyable command